src/main/java/core/basesyntax/strategy/FruitTransactionStrategy.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.strategy.impl.BalanceTransaction;
+import core.basesyntax.strategy.impl.PurchaseTransaction;
+import core.basesyntax.strategy.impl.ReturnTransaction;
+import core.basesyntax.strategy.impl.SupplyTransaction;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class FruitTransactionStrategy {
+    private static final int INDEX_TRANSACTION = 0;
+    private static final int INDEX_FRUIT = 1;
+    private static final int INDEX_QUANTITY = 2;
+    private static final String SEPARATOR = ",";
+    private FruitTransaction fruitTransaction;
+
+    public void considerTransactionWithFruit(List<String> fruitConsider) {
+        FruitTransaction fruitTransaction;

```

Comment:
make this map private satatic final field and initialise it once in static block